src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+server.port=8761
+eureka.client.register-with-eureka=false
```

Comment:
By default, when the Eureka server starts up, it will attempt to register itself with other Eureka servers (if any are configured) and fetch the registry from other Eureka servers. However, in this case, we have disabled these features because this is a standalone Eureka server and we don't need it to interact with other Eureka servers.

this is the explanation about the properties 
eureka.client.register-with-eureka=false
eureka.client.fetch-registry=false